docs/package.json

```diff
@@ -19,6 +19,7 @@
     "format": "remark --use \"remark-mdx=printWidth:80\" docs/**/*.mdx -o"
   },
   "dependencies": {
+    "@databricks/design-system": "^1.12.17",
```

Comment:
do we need to add "@emotion/react" as a dependency too? i see some build failures on the CI:

https://app.circleci.com/pipelines/github/mlflow/mlflow/42793/workflows/44b80d15-f968-4f40-9250-7963f7b5c928/jobs/130255?invite=true#step-105-2892_144